### PR TITLE
Update Russian Translation 

### PR DESCRIPTION
**Pull request by SunkyMP3G**
My GitHub is very buggy, so this PR may annihilate all the code \*shrug*
Screenshots are in discord server

### CODE REVIEW

**Comment by JojoFR1** [comment]
What is the new text you added

**Comment by SunkyMP3G** [comment]
> What is the new text you added 

Because the text on buttons is too big, I cut the "Install" in both variants, and added "Select which version to install:" After the versions' list

File: src/assets/bundles/bundle_ru.properties [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
+setting.aeyama-checkUpdate.description = Выберите, будет ли мод проверять обновления
+setting.aeyama-checkBetaUpdate.name = Проверять бета релизы
+setting.aeyama-checkBetaUpdate.description = Выберите, будет ли мод проверять бета релизы.\n[red]Требуется включить функцию "Проверять обновления"!
+
+stat.armors = Броня
+stat.weapons = Оружие
+
+aeyama.updater.name = [scarlet]\u26A0[] Устаревший мод!
+aeyama.updater.info = Ваша версия мода устарела.\n\n[gray]Установленная версия: [lightgray]v{0}[]\nПоследняя версия: [lightgray]v{1}\n[white]Выберите версию для установки:[][][]
+aeyama.updater.info-beta = Ваша версия мода устарела.\n\n[gray]Установленная версия: [lightgray]v{0}[]\nПоследняя версия: [lightgray]v{1}[]\nБета релиз: [lightgray]v{2}\n[white]Выберите версию для установки: [][][]
```
Comment (JojoFR1) [inline src/assets/bundles/bundle_ru.properties:18]: Why did you remove the `[accent]`?

Comment (SunkyMP3G) [reply src/assets/bundles/bundle_ru.properties:18]: I removed the mod's name because it felt like the game wanted to update this mod instead of mod to update itself. I can revert this
